.stylintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "colons": "never",
+  "groupOutputByFile": true,
+  "indentPref": 2,
+  "prefixVarsWithDollar": false,
```

Comment:
Isn't prefixing with $ for variables a good a good style to disambiguate vars vs css props ?

Reply:
:+1: I recall that we had an informal consensus to do so going forward but hadn't backported existing code yet. `$white` lol

Reply:
works for me, I don't have a strong preference.